x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/action/PutCalendarActionResponseTests.java

```diff
@@ -5,27 +5,40 @@
  */
 package org.elasticsearch.xpack.core.ml.action;
 
+import org.elasticsearch.client.ml.PutCalendarResponse;
 import org.elasticsearch.common.xcontent.XContentParser;
-import org.elasticsearch.test.AbstractStreamableXContentTestCase;
+import org.elasticsearch.protocol.AbstractHlrcXContentTestCase;
 import org.elasticsearch.xpack.core.ml.calendars.Calendar;
 import org.elasticsearch.xpack.core.ml.calendars.CalendarTests;
 
 import java.io.IOException;
 
-public class PutCalendarActionResponseTests extends AbstractStreamableXContentTestCase<PutCalendarAction.Response> {
+public class PutCalendarActionResponseTests extends AbstractHlrcXContentTestCase<PutCalendarAction.Response, PutCalendarResponse> {
```

Comment:
Could it extend `AbstractHlrcStreamableXContentTestCase`?  It would be nice to keep the streamability aspect of the test.